.github/workflows/tests-on-push.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         run: poetry run pytest -s test/benchmarking/
 
 
-  e2e-tests:
+  legacy-e2e-tests:
```

Comment:
By renaming the e2e into legacy-e2e, there is room for the new e2e tests with testcontainers